### ISSUE TITLE
[Upload] Undefined index

### ISSUE THREAD

**Issue by Volmarg** [Volmarg/personal-management-system#20]
![image](https://user-images.githubusercontent.com/37488679/73608047-2c721c00-45be-11ea-8b78-2f98475637f1.png)

*info*
- Only when uploading more than ~10 images
  - happened when was uploading via widget im **My images**

**Comment by Volmarg** [comment]
This issue is related to DataTable pagination.
Only data from visible page is being sent as rest is removed from dom upon page change.

Possible solutions to check:
* https://stackoverflow.com/questions/42038727/how-to-send-to-records-in-data-table-pagination-to-the-server-class
* https://datatables.net/forums/discussion/48151/how-to-turn-off-pagination
* 
````
submit.on(...{
event.prevent();
readDatatableRows();
insertTheRowsInDomAsHidden();
submit();
})

````

**Comment by Volmarg** [comment]
Fixed: 56c4a4dd8d655e2f80a0c41e7f13144cce970047
